deploy/README.md

```diff
@@ -0,0 +1,88 @@
+# inji Deployment in Kubernetes Environment
+## Overview
+* This guide will walk you through the deployment process of the Esignet application.
+* The setup involves creating
+  * Kubernetes cluster
+  * Setting up Nginx
+  * Installing Istio
+  * Configuring storage class
+  * Configuring the necessary dependent services
+  * Deploying Esignet services
+## Deployment
+### K8 cluster
+* Kubernetes cluster should be ready with storage class and ingress configured properly.
+* Below is the document containing steps to create and configure K8 cluster.
+  * __Onprem RKE CLuster__ : Create RKE K8 cluster using mentioned [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/mosip/on-prem#mosip-k8s-cluster-setup-using-rke).
+      * __Persistence__ : Setup storage class as per [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.1/mosip/on-prem#storage-classes).
+      * __Istio service mesh__ : Setup Istio service mesh using [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/mosip/on-prem#istio-for-service-discovery-and-ingress).
+      * __Nginx__ : Setup and configure nginx as per [steps](https://github.com/mosip/k8s-infra/blob/v1.2.0.2/mosip/on-prem/nginx).
+      * __Logging__ : Setup logging as per [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/logging).
+      * __Monitoring__ : Setup monitoring consisting elasticsearch, kibana, grafana using [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/monitoring).
+  * __AWS EKS cluster__ : Create AWS EKS cluster using mentioned [steps](https://github.com/mosip/k8s-infra/tree/main/mosip/aws#mosip-cluster-on-amazon-eks).
+      * __Persistence__ : Setup storage class as per [steps](https://github.com/mosip/k8s-infra/tree/main/mosip/aws#persistence).
+      * __Ingress and Loadbalancer__ : Setup nginx and configure NLB for exposing services outside using [steps](https://github.com/mosip/k8s-infra/tree/main/mosip/aws#ingress-and-load-balancer-lb).
+      * __Logging__ : Setup logging as per [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/logging).
+      * __Monitoring__ : Setup monitoring consisting elasticsearch, kibana, grafana using [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/monitoring).
+
+### Install Pre-requisites
+* `global` configmap: For inji K8's env, `global` configmap in `default` namespace contains Domain related information. Follow below steps to add domain details for `global` configmap.
+    * Copy `inji-global-cm.yaml.sample` to `inji-global-cm.yaml`.
```

Comment:
Can we remove the use of inji global config cm and instead add it as inji-stack-config and then add the same to the deployments. 
It should only contain inji related configurations needed by all the stack modules.